container-templates/docker-compose/.devcontainer/library-scripts/common-debian.sh

```diff
@@ -186,9 +186,20 @@ else
 fi
 
 # .bashrc/.zshrc snippet
-RC_SNIPPET="$(cat << EOF
-export USER=\$(whoami)
-if [[ "\${PATH}" != *"\$HOME/.local/bin"* ]]; then export PATH="\${PATH}:\$HOME/.local/bin"; fi
+RC_SNIPPET="$(cat << 'EOF'
+
+if [ -z "${USER}"]; then export USER=$(whoami); fi
```

Comment:
A missing space! I've got ```bash: [: missing `]'``` when execute `bash` in the container.

Change `[ -z "${USER}"]` into `[ -z "${USER}" ]` will fix the issue.

Reply:
Thanks! Fixed.